.github/workflows/CI_ubuntu_windows.yml

```diff
@@ -75,5 +75,5 @@ jobs:
           restore-keys: ${{ runner.os }}-gradle
 
       - name: Build with Gradle
-        run: ./gradlew.bat build --continue -x :ballerina-lang:test -x :jballerina-unit-test:test -x :jballerina-integration-test:test -x :jballerina-debugger-integration-test:test  -x :ballerina-cli:test -x :ballerina-shell:shell-cli:test -x createJavadoc --stacktrace -scan --console=plain --no-daemon --no-parallel
+        run: ./gradlew.bat build --continue -x :jballerina-unit-test:test -x :jballerina-integration-test:test -x :jballerina-debugger-integration-test:test  -x :ballerina-cli:test -x :ballerina-shell:shell-cli:test -x createJavadoc --stacktrace -scan --console=plain --no-daemon --no-parallel
```

Comment:
We need to remove cli tests also from this list. can handle from a separate PR

Reply:
It was disabled due to an intermittent failure when cleaning the target. It is tracked by https://github.com/ballerina-platform/ballerina-lang/issues/27455